tests_thien/pages/dashboard/login_page.py

```diff
@@ -0,0 +1,22 @@
+from selenpy.element.text_box import TextBox
+from selenpy.support.conditions import have
+from selenpy.support import browser
+from selenpy.element.base_element import BaseElement
+
+class DashBoardLoginPage():
+    
+    _txt_username = TextBox("id=username")
+    _txt_password = TextBox("id=password")
+    _btn_login = BaseElement("class=btn-login")
+    
+    def __init__(self):
+        pass
+    
+    def open_dashboard(self):
+        browser.open_url("http://192.168.188.99/TADashboard/login.jsp")
```

Comment:
Don't hard the url here. Define it as variable

Reply:
updated